debug-helpers/debug-helper.js

```diff
@@ -0,0 +1,116 @@
+mapboxgl.accessToken = "pk.eyJ1IjoieWNob3kiLCJhIjoiY2pmOTYwdzZ5MG52dDJ3b2JycXY4ZDU5ciJ9.m9H_Mqu1b42AObg_u_tjpA";
```

Comment:
Should this be here or in another config file that is not going to the repo?

Reply:
Hmmm... since the editor is running off of node.js, we could hide the API key in the .env file

Reply:
yes, we should hide the api key. also for the js/main.js perhaps, it is also there.

Reply:
I agree this key should not be here. `.env` is ok, but keep in mind this should all be easy to run, so if you do add a `.env`, then add a default `.env` file so that future devs know how to use it.

Reply:
I created a new issue for moving the keys to env variables.